doc/guides/kconfig/index.rst

```diff
@@ -909,6 +909,12 @@ the above to look like:
    config FLASH_BASE_ADDRESS
       default 0x20010000
 
+Cmake related Functions
+=======================
+
+.. code-block:: none
+
+  cmake_in_list(kconf, _, elem, list):
```

Comment:
Indent by three spaces, extra `:` at the end.

Reply:
Done, but no more aligned with other "Devicetree Related Functions"